index.php

```diff
@@ -124,6 +124,12 @@
         }
     }
 
+    // redirect to ticket
+    if (isset($_GET["redirect"])) {
+        Toolbox::manageRedirect($_GET["redirect"]);
+        echo '<input type="hidden" name="redirect" value="'.Html::entities_deep($_GET['redirect']).'"/>';
```

Comment:
This input is already output by template. Only the `Toolbox::manageRedirect()` logic is required here.
```suggestion
```